### ISSUE TITLE
buffer for 30s every 30s

### ISSUE THREAD

**Issue by MichaelSaunders5** [yuliskov/SmartTube#2182]
I have a 4-month-old Google TV.   After the last update, SmartTube stops every 30s to buffer for 30s.  I tried the lowest video settings and the highest buffer and can't get better than that.  Beta has the same problem as stable.  I reset the Wifi and got a new modem.  No change.  Is this something YouTube is doing to hobble SmartTube?

**Comment by ZeroOnions** [comment]
Probably, I have the same problem, but only with some videos. And now some videos won't load at all, but they do in the original app, so it's certainly not network issue.

**Comment by LeCommonMan** [comment]
Facing similar issue on fire TV,
Few videos refuses to load and by chance if they,... load keep on buffering after a brief playback.

**Comment by p-money666** [comment]
Me too! This happened a few months ago and I thought it was my internet causing issues but everything else was loading fine.

Seems to be the same issue again on the latest beta. I'm on a fast hardwired connection so I'm not expecting any buffering.

I'm just thinking for reasons like this is it better to stay on the stable builds? But I note that the beta is recommended!

**Comment by Sixotoo** [comment]
I’m also getting this on fire stick 4k

**Comment by ZeroOnions** [comment]
It doesn't matter, I never used beta version, and the issue appeared nevertheless. Then, after roughly two days it disappeared for me and all videos load just fine.

This is google doing *something* and devs of this app not doing anything to counter that. They don't seem to give a damn about those reported issues anyway - they never respond to anyone.

**Comment by p-money666** [comment]
Yeah I'll wait for the next beta release to see if anything's improved otherwise I might need to bite the bullet and go for YouTube Premium (can get it for cheaper using a VPN).

**Comment by ZeroOnions** [comment]
I'll miss the video quality setting, but yeah - I might bite the bullet either. Not worth the hassle to save just a few bucks

**Comment by torianemoia** [comment]
Was having the same issue where videos would buffer constantly and some would refuse to load despite internet working fine. prayed to multiple deities from various faiths last night. Issue resolved.

**Comment by steve51184** [comment]
Same issue but I found dropping from 4k to 2k fixes the issue

**Comment by Chealer** [comment]
@MichaelSaunders5 : Does this persist with a current version? And if so, which videos are affected?
